src/main/webapp/resources/js/apis/projects/reference-files.js

```diff
@@ -8,7 +8,7 @@ import { setBaseUrl } from "../../utilities/url-utilities";
  */
 export async function getProjectReferenceFiles(projectId) {
   return await axios
-    .get(setBaseUrl(`/projects/${projectId}/settings/ajax/reference/all`))
+    .get(setBaseUrl(`ajax/reference-files/${projectId}/all`))
```

Comment:
Same as your controller method, do we really need the `/all` at the end?  What might be a better pattern here is `/ajax/reference-files?projectId=4` or something similar.  That way we know exactly when we write that url that is for a project.  Food for thought.

Reply:
Nope that's a good idea. Updated in b0250f63048872cf049ac1a2c4d98926b2ed6072